project/Build.scala

```diff
@@ -316,6 +315,8 @@ object Build {
     parallelExecution in Test := false,
 
     genDocs := Def.inputTaskDyn {
+      IO.write(file("./docs/_site/last-nightly"), nightlyVersion)
```

Comment:
"latest-nightly"?

Reply:
Are the docs only updated on nightly release? Will there always be a corresponding published version on sonatype?

Reply:
Hum, you're right this is wrong. The doc is updated on every PR merged on master. So the version will not be published on sonatype yet. @smarter Should we publish doc only on nightly?

Reply:
You guys are coming to all the right conclusions :tada: 

It indeed needs to be the latest nightly version published. Maybe maven has a query API for this?

Reply:
The latest base version would do for my needs, we could use that to query maven-metadata.xml for the latest release here https://github.com/lampepfl/dotty/blob/58ca93df0caadd238e7450ae60f8044ecc0ad9ef/sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala#L17